source/index.html.md

```diff
@@ -0,0 +1,429 @@
+---
+title:TPMS API Reference
+
+language_tabs: # must be one of https://git.io/vQNgJ
+  - shell
+ 
+
+toc_footers:
+  - <a href='#'>Sign Up for a Developer Key</a>
+  - <a href='https://github.com/lord/slate'>Documentation Powered by Slate</a>
+
+includes:
+  - errors
+
+search: true
+---
+
+# Introduction
+
+Welcome to the Turntabl Project Management System (TPMS) API. This API documentation contains TPMS API endpoints with regards to Time-entry.
+
+The languages used for TPMS Time entry application are Psql, Java and Angular.
+ 
+
+Some of the technologies used are SAML, Stored Procedure, Aws RDS, etc.
+
+The Application contains various services: Project Service, Employee Service and Logged Hours Service which use same Database.
+
+The Project Service has various functionalities such as add project, list all project, get project by employee ID and delete project
+
+
+
+
+# User 
+```shell
+#FORMAT OF DATA 
+```
+Represents user details.
+
+## User attributes:
+>
+Employee data format:
+>
+
+```json
+
+  {
+  "employee_id": 0,
+  "employee_firstname": "string",
+  "employee_lastname": "string",
+  "employee_email": "string",  
+ } 
+```
+
+
+Name | Type | Description
+---------|---------- | -----------
+employee_id | Number | unique identifier for employee. 
+employee_firstname | String | First Name. 
+employee_lastname | String | Last Name.
+employee_email | String | Email.
+
+
+> 
+Project data format:
+>
+```json
+ {
+  "project_id": 0,
+  "project_date": "date",
+  "project_hours": "string",
+  "sick_date": "date",
+  "vacation_date": "date",
+ }
+```
+
+
+
+Name |Type| Description
+--------- | ----------- | -----------
+project_id | Number | unique identifier for project
+project_date | Date | Date of project
+project_hours | Number | Hours.
+
+
+Name |Type | Description
+--------- | ----------- | -----------
+sick_date | Date | Date for sick log.
+vacation_date | Date | Date for vacation log.
+
+
+
+# Authentication
+
+> To authorize, use this code:
+
+
+```shell
+curl "https://project.services.turntabl.io/" 
+```
+
+> Make sure you are a Turntabl employee before assess this API.
+
+TPMS uses SAML to allow access to the API. You can only be registered a new Turntabl employee by the Administrator, TPMS API at our [developer portal](http://turntabl.io/developers).
```

Comment:
Don't confuse the TPMS application with the backing services. User access to the application is authenticated with GSuite SAML.